oleaut32.go

```diff
@@ -187,6 +198,182 @@ func VariantBSTRToString(value *VAR_BSTR) string {
 	return BSTRToString(value.bstrVal)
 }
 
+func (v *VARIANT) LVal() int32 {
+	value, err := v.GetLVal()
+	if err != nil {
+		log.Fatal(err)
+	}
+	return value
+}
+
+func (v *VARIANT) GetLVal() (int32, error) {
+	if v.Vt != VT_I4 {
+		return 0, fmt.Errorf("Error: LVal() v.Vt !=  VT_I4, ptr=%p, value=%+v", v, v)
+	}
+	p := (*VAR_I4)(unsafe.Pointer(v))
+	return p.lVal, nil
+}
+
+func (v *VARIANT) SetLVal(value int32) {
+	v.Vt = VT_I4
+	p := (*VAR_I4)(unsafe.Pointer(v))
+	p.lVal = value
+}
+
+func (v *VARIANT) ULVal() uint32 {
+	value, err := v.GetULVal()
+	if err != nil {
+		log.Fatal(err)
+	}
+	return value
+}
+
+func (v *VARIANT) GetULVal() (uint32, error) {
+	if v.Vt != VT_UI4 {
+		return 0, fmt.Errorf("Error: LVal() v.Vt !=  VT_UI4, ptr=%p, value=%+v", v, v)
+	}
+	p := (*VAR_UI4)(unsafe.Pointer(v))
+	return p.ulVal, nil
+}
+
+func (v *VARIANT) SetULVal(value uint32) {
+	v.Vt = VT_UI4
+	p := (*VAR_UI4)(unsafe.Pointer(v))
+	p.ulVal = value
+}
+
+func (v *VARIANT) BoolVal() VARIANT_BOOL {
```

Comment:
This should be called `MustBool` and use `panic` instead of `log.Fatal` to allow recovery.